### ISSUE TITLE
Update Project from database creates objects in in correct folders

### ISSUE THREAD

**Issue by jasonhorner** [microsoft/azuredatastudio#25116]
Type: <b>Bug</b>

When using the **Update Project from Database option** (and possibly schema compare)....  objects that don't exist will be created in project folders without spaces in the names. 

![image](https://github.com/microsoft/azuredatastudio/assets/7981510/8e397474-86bf-41f6-b7b7-9d28a53b9e55)

For example: if I'm missing an external table. but there is already a folder for external tables called "External Tables" it will create the new objects under a new folder called "ExternalTables". 

![image](https://github.com/microsoft/azuredatastudio/assets/7981510/f3dd7a3e-9f05-46e4-a3b0-7f80cef88b7d)

There doesn't seem to be a way to "fix" this behavior but it isn't desired. I believe the correct behavior should be to place the objects in the existing folder. If no folder exists it should default to the same name as visual studio or SSMS would use which is "External Tables"

![image](https://github.com/microsoft/azuredatastudio/assets/7981510/db5b9171-594f-42ef-8155-b17cb67b6644)

I have confirmed the is an issue for any folder that should have a space in the name.

Extension version: 1.4.1
Azure Data Studio version: azuredatastudio 1.47.0 (c7c2b2f21505562d21972d4c135119d00806db4f, 2023-11-07T17:17:45.669Z)
OS version: Windows_NT x64 10.0.19045
Restricted Mode: No
Preview Features: Enabled
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.60GB (15.81GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by SakshiS-harma** [comment]
@jasonhorner Thanks for bringing this issue to light. Did you notice this issue pop up recently after the update or has this been an issue for long now?

**Comment by jasonhorner** [comment]
@SakshiS-harma  I don't think it is a regression.. I have noticed it for both serverless projects and traditional sql projects...
but not 100% sure

**Comment by SakshiS-harma** [comment]
@jasonhorner Wondering if you have tested it with "Create project from database"? Do you see the same behavior there as well?

**Comment by jasonhorner** [comment]
@SakshiS-harma  yes, the behavior is the same when creating a new project from a database
